src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -1445,7 +1442,6 @@ while getopts "a:b:cd:e:f:g:h:i:j:m:p:r:s:t:v:w:x:y:z:" OPTION; do
         d)  SDISTS="$OPTARG"                  ;;
         e)  pre_bootstrap_0 "$OPTARG"         ;;
         f)  FORWARD_TUNNEL_ENDPOINT="$OPTARG" ;;
-        g)  VIRTENV_DIST="$OPTARG"            ;;
```

Comment:
really the last one thing - L1440 option character `g` and its description in L1420

Reply:
thanks! :)